### ISSUE TITLE
List of clusters limited to 100

### ISSUE THREAD

**Issue by shivasfotedar** [keidarcy/e1s#226]
1. List of ecs clusters visible in the UI is limited to 100. In scenarios where more than 100 clusters are there in the account , cluster is not shown in the UI.

Additional points that can be addressed:

1. Search functionality in UI similar to k9s
2.  An argument for the cluster name while using e1s command to open a specific cluster like one we have for k9s.

**Comment by haidaraM** [comment]
Thanks for your great tool. I would also like search functionality. I was also thinking of filtering the clusters via the configuration file. This will avoid searching for the same clusters again and again if you are also interested in a few :-) 

Example:  Only display clusters matching the following patterns by default
```yaml
filters:
  names:
     - dev-*
     - *-playground
     - *team*
```

**Comment by keidarcy** [comment]
@shivasfotedar Latest version would support clusters more than 100 and --cluster option for the default cluster.

**Comment by keidarcy** [comment]
@haidaraM I don't think the filters feature is scalable, recommend `?` to use more possible navigation methods.

**Comment by shivasfotedar** [comment]
I encountered this issue while navigating from cluster to services in the latest release i.e. 1.35.0
`💥 operation error ECS: DescribeServices, https response error StatusCode: 400, RequestID: 0824fdb3-8739-4393-bfab-030cbb48b3db, InvalidParameterException: service names can have at most 10 items.`

**Comment by keidarcy** [comment]
@shivasfotedar this should be fixed by v1.0.36 😄
